lib/Model/BanMapper.php

```diff
@@ -28,12 +28,12 @@ public function __construct(IDBConnection $db) {
 	/**
 	 * @throws DoesNotExistException
 	 */
-	public function findForActorAndRoom(string $actorId, string $actorType, int $roomId): Ban {
+	public function findForBannedActorAndRoom(string $actorId, string $actorType, int $roomId): Ban {
 		$query = $this->db->getQueryBuilder();
 		$query->select('*')
 			->from($this->getTableName())
-			->where($query->expr()->eq('actor_id', $query->createNamedParameter($actorId, IQueryBuilder::PARAM_STR)))
-			->andWhere($query->expr()->eq('actor_type', $query->createNamedParameter($actorType, IQueryBuilder::PARAM_STR)))
+			->where($query->expr()->eq('banned_id', $query->createNamedParameter($actorId, IQueryBuilder::PARAM_STR)))
+			->andWhere($query->expr()->eq('banned_type', $query->createNamedParameter($actorType, IQueryBuilder::PARAM_STR)))
```

Comment:
Maybe it's just my personal preference, but is `banned_id` and `banned_type` really self explanatory ? Wouldn't `banned_actor_id` and `banned_actor_type` make more sense? Or too long?

Reply:
That would work. we are currently considering whether we should change actor_type, … to moderator_type, … as both devs made a mistake with that already.
But could also then just go for banned_actor_type, … and moderator_actor_type, …